### DISCUSSION TITLE
Lange Ladezeiten im Listing unter Chrome

### DISCUSSION THREAD

**Discussion by 5p4jk** [2Abendsegler/GClh#2172] [Questions]
Seit der neusten Aktualisierung lädt das Programm bei Chrome ewig lange. Woran liegt das?

**Comment by 2Abendsegler** [comment]
Moin,

auf welcher Seite bist du denn? 
Woran erkennst du, dass das Programm ewig lange lädt?

**Comment by 5p4jk** [comment]
Hallo,danke, dass du so schnell auf meine Anfrage reagierst. Das Problem habe ich auf den Seiten von GC.com.Öffne ich einen Geocache, dauert das Laden der Zusatzfunktionen, wie z.B. die VIP- Liste, das Suchfeld für die GC-Nummern, oder das farbliche Markieren der geänderten Koordinaten sehr lange. Schalte ich den Little Helper aus, ist die Seite schnell geladen. Ich möchte aber ungern auf die liebgewonnenen Zusatzfunktionen verzichten.Viele Grüße Am 10.08.2022 16:25 schrieb 2Abendsegler ***@***.***>:

Moin,

auf welcher Seite bist du denn?

Woran erkennst du, dass das Programm ewig lange lädt?

**Comment by 2Abendsegler** [comment]
Wir haben mit der letzten Version eigentlich keine Anpassungen gemacht, die im Cache Listing für solche Veränderungen sorgen könnten. Ich habe das auch gerade mal ausprobiert unter Chrome mit der Vorgängerversion und der aktuellen. Ein Cache Listing mit 3700 Logs ist bei mir in weniger als zwei Sekunde geladen.

Bei Listings mit vielen Logs kann es schon etwas dauern, weil alle Logs geladen werden müssen, bis beispielsweise die VIP Liste korrekt aufgebaut werden kann. 

Bitte mal folgendes versuchen.

1. Kannst du bitte mal folgendes Cache Listing aufrufen, dort sind nur 11 Logs enthalten, entsprechend sollte die VIP Liste quasi sofort zur Verfügung stehen. https://coord.info/GC9GTT7

2. Kannst du bitte außerdem mal die Tasten Ctrl, Shift und C gleichzeitig drücken, wenn du im Cache Listing bist. Dort dann auf den Reiter "Console" klicken. Falls dort etwas in rot eingefärbt steht, dann bitte einen Screenshot davon machen und hier posten. 

3. Und bitte deaktiviere mal alle deine Add Ons und Scripte bis auf den GClh und versuch es dann noch einmal.

**Comment by capoaira** [comment]
Moin @5p4jk,
haben die Schritte von @2Abendsegler etwas ergeben?
Könntest du bitte mal gucken, ob auf der Drafts Seite diese Probleme auch auftauchen?
Wir haben ein ähnliches/gleiches issue #2173 
Kann es sein, das dein GC Nuzername Sonderzeichen enthält?
Wir sind gerade im Prozess den Fehler zu finden leider geht das in diesem Fall nicht ohne die Mithilfe derer, die das Problem haben. Wir würden uns freuen, wenn du uns antwortest.

**Reply by Yannick-XY** [reply]
Hallo zusammen, 
auch ich hab das Problem, dass das Skript im Vivaldi-Browser jedes Mal eine Ewigkeit lädt, bis die Logs auftauchen und die Drafts-Seite ist ähnlich langsam.
Bei Aufruf des o.g. Cache-Listings lädt die Seite bei ausgeschaltetem Skript und auch sonst nichts aktivem quasi sofort. 
Mit aktiviertem GClh dauert es gut 8 Sekunden bis dann die Logs laden...

**Reply by 2Abendsegler** [reply]
Moin,
wenn du einen AdBlocker hast, dann versuch bitte mal `https://consent.cookiebot.com/uc.js` zu sperren.
[Hier](https://github.com/2Abendsegler/GClh/issues/2173#issuecomment-1229435222) ist das beschrieben. Das könnte auch für dich ein Workaround sein bis wir eine echte Lösung haben. Ich habe bereits mehrere Male erfolglos versucht, das Problem nachzustellen.
Hast du vielleicht auch das Add On "Microsoft S/MIME 20.21.531.1" im Einsatz?
